### PR TITLE
document known errors for different director versions 

### PR DESCRIPTION
**Pull request by rndmh3ro**
and [use correct var for integration test](https://github.com/telekom-mms/ansible-collection-icinga-director/commit/756ef9e77a73876674cf3b231763d88a759a64d7)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -192,6 +192,30 @@ fatal: [localhost]: FAILED! => {"changed": false, "msg": "bad return code while
...
+### Director 1.9.0
+
+When creating service checks, the tasks fails ([see](https://github.com/telekom-mms/ansible-collection-icinga-director/issues/160)):
+
+```
+failed: [icinga2-master1.localdomain] (item={'name': 'director-generic-service', 'enable_active_checks': True, 'enable_event_handler': True, 'enable_flapping': True, 'enable_notifications': True, 'eanble_passive_checks': True, 'enable_perfdata': True, 'use_agent': True, 'volatile': False}) => {"ansible_loop_var": "item", "changed": false, "item": {"eanble_passive_checks": true, "enable_active_checks": true, "enable_event_handler": true, "enable_flapping": true, "enable_notifications": true, "enable_perfdata": true, "name": "director-generic-service", "use_agent": true, "volatile": false}, "msg": "bad return code while creating: 422. Error message: Trying to recreate icinga_service (\"{\"object_name\":\"director-generic-service\"}\")"}
+```
+
+### Director 1.10.0
+
```
Comment (schurzi) [inline README.md:206]: one empty line too much

File: README.md [hunk 2]
```diff
@@ -192,6 +192,30 @@ fatal: [localhost]: FAILED! => {"changed": false, "msg": "bad return code while
 failed: [localhost] => {"ansible_loop_var": "item", "changed": false, "item": "localhost", "msg": "AbstractDigestAuthHandler does not support the following scheme: 'Negotiate'", "status": -1, "url": "https://icinga-director.example.com/director/host?name=foohost"}
 ```
 
+## Known Errors with different Director versions
+
+### Director 1.9.0
```
Comment (schurzi) [inline README.md:197]: I'm unsure if it would be better to reverse the order, so the most current version always is first.

**Review by schurzi — APPROVED** [review]
I really like, that we are linking to our own issues here, so we can add more information independently

File: README.md [hunk 3]
```diff
@@ -192,6 +192,30 @@ fatal: [localhost]: FAILED! => {"changed": false, "msg": "bad return code while
...
+Existing service apply rule objects cannot be modified ([see](https://github.com/telekom-mms/ansible-collection-icinga-director/issues/190)):
+
+```
+TASK [Add service apply rule to icinga] ********************************************************************************************
+fatal: [localhost]: FAILED! => {"changed": false, "msg": "exception when deleting: 'id'"}
+```
+
+### Director 1.11.0
+
+You cannot create usergroups because of invalid property assign_filter ([see](https://github.com/telekom-mms/ansible-collection-icinga-director/issues/228)).
```
Comment (schurzi) [inline README.md:217]: Is is UserGroup or HostGroup?

Comment (rndmh3ro) [reply README.md:217]: Thanks for noticing! The problem is with UserGroups. I changed the issue-title accordingly.

**Pull request merged**